packages/javascript-kernel/src/worker.ts

```diff
@@ -44,16 +45,16 @@ export class JavaScriptRemoteKernel {
   async execute(content: any, parent: any) {
     const { code } = content;
     try {
-      const result = self.eval(code);
+      const result = self.eval(code) as unknown;
       this._executionCount++;
 
       const bundle = {
         data: {
-          'text/plain': result,
+          'text/plain': inspect(result),
         },
         metadata: {},
         execution_count: this._executionCount,
-      };
+      } satisfies KernelMessage.IExecuteResultMsg['content'];
       postMessage({
         bundle,
         type: 'execute_result',
```

Comment:
there is a typesafe generic factory function `KernelMessage.createMessage()` which would be useful throughout this module. They won't work directly with the language currently spoken over `postMessage()`, but we control both ends and can change it to accommodate them